api/src/main/java/org/openmrs/module/fhir/api/util/FHIRUtils.java

```diff
@@ -99,6 +99,10 @@ public static String getLocationStrategy() {
 		return Context.getAdministrationService().getGlobalProperty("fhir.location.locationStrategy");
 	}
 
+	public static String getObservationStrategy() {
+		return Context.getAdministrationService().getGlobalProperty("fhit.observation.observationStrategy");
```

Comment:
There is a typo. Please change fhit -> fhir

Reply:
Nice catch, thanks!